example/z-docker-opam/.dockerignore

```diff
@@ -0,0 +1,39 @@
+# ocamlbuild working directory
+_build/
+_esy/
+
+# ocamlbuild targets
+*.byte
+*.native
```

Comment:
IMO it's best to skip this. People won't be using Ocamlbuild in their own build, realistically. The same goes for `_esy` for an OCaml project. Also, for the sake of an example (so, to focus on only the most important parts), I wouldn't bother excluding `.merlin` and the like. I would probably even recommend a `.dockerignore` that excludes *eveything*, but then opts back into e.g. `src/` with `!`. In any case, I think the `.dockerignore` should become smaller for an example.

Reply:
Done, I kept the default and listed the exclusion one by one, but it only contains the strict necessary now

Reply:
I recommend to add `_opam` back in, since that *will* come up in most opam projects, and the only reason it's not already an issue in the example is that it is in a monorepo :)

Reply:
Done